selftests/unit/test_qemu_devices.py

```diff
@@ -825,7 +826,7 @@ def test_qdev_functional(self):
 
     def test_qdev_hotplug(self):
         """ Test the hotplug/unplug functionality """
-        qdev = self.create_qdev('vm1', False, True)
+        qdev = self.create_qdev('vm1', 'pc', False, True)
```

Comment:
Tests have to be updated together with the actual change, otherwise you'll get failures after the first commit...

Reply:
OK, will combine them.